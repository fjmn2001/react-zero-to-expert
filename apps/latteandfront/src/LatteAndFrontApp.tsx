const LatteAndFrontApp = () => {
  return (
    <div className="App">
      <header className="App-header">
        <h1>
          <span role="img" aria-label="Yellow heart">
            ðŸ’›
          </span>{" "}
          Latte and Front
        </h1>
      </header>
    </div>
  )
}
export default LatteAndFrontApp
