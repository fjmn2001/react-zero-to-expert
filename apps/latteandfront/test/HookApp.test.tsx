import { describe, expect, test } from "vitest"
import { render } from "@testing-library/react"
import LatteAndFrontApp from "../src/HookApp"

describe("Testing HookApp.tsx", () => {
  test("it should show correctly", () => {
    const { container } = render(<LatteAndFrontApp />)
    expect(container).toMatchSnapshot()
  })
})
