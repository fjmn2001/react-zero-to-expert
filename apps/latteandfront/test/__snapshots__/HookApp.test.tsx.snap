// Vitest Snapshot v1

exports[`Testing LatteAndFrontApp.tsx > it should show correctly 1`] = `
<div>
  <h1>
    Hi there!
  </h1>
</div>
`;
